.openshift-ci/ci_tests.py

```diff
@@ -57,10 +58,10 @@ class OperatorE2eTest(BaseTest):
     SCORECARD_TEST_TIMEOUT_SEC = 20 * 60
 
     def __init__(self):
-        self.test_outputs = [
-            "operator/build/kuttl-test-artifacts",
-            "operator/build/kuttl-test-artifacts-upgrade",
-        ]
+        self.test_results = {
+            "operator/build/kuttl-test-artifacts": "kuttl-test-artifacts",
```

Comment:
I think it might be better to have the dict the other way, since we primarily want to make sure the destination name is unique..